bciers/apps/registration/app/bceidbusiness/industry_user_admin/dashboard/operations/[operationId]/[formSection]/page.tsx

```diff
@@ -0,0 +1,18 @@
+// 🚩 flagging that for shared routes between roles, "Page" code is a component for code maintainability
+
+import { Button } from "@mui/material";
+import Link from "next/link";
+
+export default async function Page(params: any) {
+  console.log("params.params.operationId", params.params.operationId);
+  return (
+    <div>
+      TBD
+      <Link
+        href={`/dashboard/operations/${params.params.operationId}/facilities/new`}
+      >
+        <Button variant="contained">Add Facility</Button>
```

Comment:
This file will get replaced; just stuck a button on this page for now for easier navigation/testing